osbs/build/spec.py

```diff
@@ -159,7 +159,8 @@ class ProdSpec(CommonSpec):
     kojiroot = BuildParam("kojiroot", allow_none=True)
     kojihub = BuildParam("kojihub", allow_none=True)
     image_tag = BuildParam("image_tag")
-    source_secret = BuildParam("source_secret", allow_none=True)
+    pulp_secret = (BuildParam("pulp_secret", allow_none=True) or
+                   BuildParam("source_secret", allow_none=True))
```

Comment:
The `or` here does nothing because the left hand side always evaluates to true ... or am I missing something?